website/conferences/views.py

```diff
@@ -233,12 +233,8 @@ def conference_submissions(**kwargs):
         # For efficiency, we filter by tag first, then node
         # instead of doing a single Node query
         projects = set()
-        for tag in Tag.find(Q('_id', 'iexact', conf.endpoint)):
-            for node in tag.node__tagged:
-                if not node:
-                    continue
-                if not node.is_public or node.is_deleted:
-                    continue
+        for tag in Tag.find(Q('lower', 'eq', conf.endpoint)):
```

Comment:
I don't think `conf.endpoint` is guaranteed to be lower case here; better to use `.lower()` just in case.